src/aind_ephys_rig_qc/qc_figures.py

```diff
@@ -190,6 +190,7 @@ def plot_drift(directory, stream_name, block_index=0):
         "vmax": 0,
         "cmap": "Greys_r",
         "figsize": [10, 10],
+        "phase_shift": {"margin_ms": 100.0},
```

Comment:
maybe you can make `phase_shift` optional:
- if `"phase_shift"` is `None`, don't run it
- else, run it with params

Reply:
Fixed in the new version by adding an editable .json file.